### ISSUE TITLE
Confusing EIP output

### ISSUE THREAD

**Issue by robertdfrench** [robertdfrench/zoolander-ci#53]
```console
Outputs:

ip = {
  "dns" = "zoolander-ci.robertdfrench.com"
  "id" = "eipalloc-039b7c92ec580ee1b"
  "ip" = "52.6.63.33"
}
zoolander = 52.4.19.86
```

So which is the right IP address?

**Comment by robertdfrench** [comment]
And the EIP gets destroyed *every* time I run terraform, so something is amok here.

**Comment by robertdfrench** [comment]
This is fixed in 730066dd2c2efb7603376f84a4724ebeb29c1d9a
